### PR TITLE
Fix circular dependency

### PR DESCRIPTION
**Pull request by tnorling**
Fixes circular dependency

### CODE REVIEW

**Review by derisen — APPROVED** [review]
LGTM -not sure if needs action but the same pattern seems to exist in node-token-validation as well

File: lib/msal-node/src/config/Configuration.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ const DEFAULT_LOGGER_OPTIONS: LoggerOptions = {
 
 const DEFAULT_SYSTEM_OPTIONS: Required<NodeSystemOptions> = {
     loggerOptions: DEFAULT_LOGGER_OPTIONS,
-    networkClient: NetworkUtils.getNetworkClient(),
+    networkClient: new HttpClient(),
```
Comment (sameerag) [inline lib/msal-node/src/config/Configuration.ts:125]: I thought the initial idea was to ensure generic client setup, where `setNetworkClient()` would enable them to set the specific network client.

**Review by sameerag — APPROVED** [review]
Approved. I think we already have a demo by Robbie for custom network implementation, so this looks fine.

**Comment by ghost** [comment]
:tada:`@azure/msal-node@v1.14.4` has been released which incorporates this pull request.:tada:

Handy links:
* [Release Notes](https://github.com/AzureAD/microsoft-authentication-library-for-js/releases/tag/msal-node-v1.14.4)
* [Official NPM Release](https://www.npmjs.com/package/@azure/msal-node/v/1.14.4)

**Pull request merged**